### DISCUSSION TITLE
Unable to use my Azure Credits

### DISCUSSION THREAD

**Discussion by kajallochab** [microsoft/SATechnicalOnboarding#69] [Questions]
I am an Alpha MLSA, and I redeemed my credits, but they are not reflected in my code space or Azure account. I even received the confirmation for having activated my benefits. Please help me solve this as soon as possible. I have attached all the screenshots.

<img width="1290" alt="Screenshot 2023-11-04 at 7 54 28 PM" src="https://github.com/microsoft/SATechnicalOnboarding/assets/109052494/f8d495f7-49bb-4040-a860-ce8b724963db">

<img width="1297" alt="Screenshot 2023-11-04 at 7 54 38 PM" src="https://github.com/microsoft/SATechnicalOnboarding/assets/109052494/4ba03a76-484e-4cc6-a9b3-3b72961018a0">

<img width="1290" alt="Screenshot 2023-11-04 at 7 54 28 PM" src="https://github.com/microsoft/SATechnicalOnboarding/assets/109052494/fabc9170-0a94-46b2-bfc0-461032d3a3c4">

<img width="1297" alt="Screenshot 2023-11-04 at 7 54 38 PM" src="https://github.com/microsoft/SATechnicalOnboarding/assets/109052494/6074a9f0-f17d-41bd-894f-63f5687fd4c4">

<img width="1001" alt="Screenshot 2023-11-04 at 7 55 55 PM" src="https://github.com/microsoft/SATechnicalOnboarding/assets/109052494/77b43a7d-af7e-47ba-938c-1371a376f2e1">

![Screenshot 2023-11-04 at 7 59 45 PM](https://github.com/microsoft/SATechnicalOnboarding/assets/109052494/b60ea51c-659c-4291-b850-7bce2a92e49d)

**Comment by AnthonyByansi** [comment]
Hello, @kajallochab. Please ensure that you have signed into your MLSA Azure account after installing the [Azure Account VS Code Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account). This extension will display the subscriptions associated with your account, which you can utilize to complete the technical onboarding process. Thank you!

**Reply by kajallochab** [reply]
I have done that as well, and I am still unable to utilize it. 

<img width="553" alt="Screenshot 2023-11-05 at 12 00 12 AM" src="https://github.com/microsoft/SATechnicalOnboarding/assets/109052494/a033417f-df96-4801-be57-660d7e8f9259">

**Reply by kajallochab** [reply]
I cannot even see the subscription on my Azure account, but as you can see from the previous screenshots, I have redeemed the credits correctly.

<img width="1304" alt="Screenshot 2023-11-05 at 12 01 40 AM" src="https://github.com/microsoft/SATechnicalOnboarding/assets/109052494/cbafb419-cea9-4fac-92b5-491c54550cdc">

**Reply by AnthonyByansi** [reply]
Hello @kajallochab,

If you have successfully redeemed your Azure monthly credits but are unable to see the corresponding subscription, I would advise you to reach out to the SA-coordinator for assistance in resolving this matter.

**Reply by Rudra6929** [reply]
see u have to install vs code them install  azure vs code extension  them open ur command palette by pressing ctrl+shift+p them type >Azure: Sign In after succesfully signed in go and check your azure. hope this helps
